src/app/public/modules/auth-http-client/no-auth-interceptor.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  Injectable
+} from '@angular/core';
+
+import {
+  HttpEvent,
+  HttpHandler,
+  HttpInterceptor,
+  HttpRequest
+} from '@angular/common/http';
+
+import {
+  SkyAppConfig
+} from '@skyux/config';
+
+import {
+  Observable
+} from 'rxjs/Observable';
+
+import {
+  SKY_AUTH_PARAM_AUTH
+} from './auth-interceptor-params';
+
+import {
+  BBAuthClientFactory
+} from '@skyux/auth-client-factory';
+
+import 'rxjs/add/observable/fromPromise';
+
+import 'rxjs/add/operator/switchMap';
+
+@Injectable()
+export class NoAuthInterceptor implements HttpInterceptor {
```

Comment:
The class name needs to have a `Sky` prefix to match our naming conventions (i.e. `SkyNoAuthInterceptor`).

Reply:
Adding the `Sky` prefix sounds good.